scripts/seed_products.js

```diff
@@ -0,0 +1,33 @@
+// To access your database
+import { db } from '$api/src/lib/db'
+import { dummyPrices } from './dummy/prices'
+
+require('dotenv').config()
+const stripe = require('stripe')(process.env.STRIPE_SK)
+
+export default async ({ args }) => {
+  const priceResults = []
+  let c = 0
+
+  // Create once-off prices
+  for (const price of dummyPrices) {
+    try {
+      const result = await stripe.prices.create(price)
+      priceResults.push(result)
+    } catch (err) {
+      console.log(err.raw.message)
+      priceResults.push({ error: err.raw.message })
+      // if product exit then escapes the loop
```

Comment:
Tried to expand this a bit to what I thought you meant (mostly just "products" -> "prices"):
```suggestion
      // If any of the price creations fail, escape the loop
```